app/templates/cp_nigeria/steps/step_progression.html

```diff
@@ -24,17 +24,12 @@ <h1 class="wizard__name">
         <div class="wizard__steps">
             <ul>
                 {% for step in step_list %}
-
                 <li {% if step_id == forloop.counter %} class="active" {% endif %}>
```

Comment:
I ammended the commit where you had made all li items with class "active" as one would not see which one was the selected one anymore. All the li items are clickable because you already removed the `if forloop.counter < max_step`